<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaosIsp</class>
 <widget class="QMainWindow" name="CaosIsp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CoActionOS ISP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="Installer" name="installTab">
       <attribute name="title">
        <string>Install</string>
       </attribute>
      </widget>
      <widget class="IspTerminal" name="terminalTab">
       <attribute name="title">
        <string>Terminal</string>
       </attribute>
      </widget>
      <widget class="Preferences" name="preferencesTab">
       <attribute name="title">
        <string>Preferences</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="labelTab">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Label</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="CConnect" name="connectWidget" native="true"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>8</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>8</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CConnect</class>
   <extends>QWidget</extends>
   <header>CSdk/CConnect.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IspTerminal</class>
   <extends>QWidget</extends>
   <header>IspTerminal.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Preferences</class>
   <extends>QWidget</extends>
   <header>Preferences.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Installer</class>
   <extends>QWidget</extends>
   <header>Installer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
